### ISSUE TITLE
ItemAdder Prefix

### ISSUE THREAD

**Issue by FilexCZE** [kangarko/ChatControl#1305]
Minecraft Version: 1.7.1
MySQL: true
BungeeCord: false

Hey, i haved on my server ItemAdder, i testing prefix by this plugin
![image](https://user-images.githubusercontent.com/75536352/132244762-f51cf8ce-62eb-4fca-b3d8-8362e488af11.png

But i haved problem, if i wrote to chat ![image](https://user-images.githubusercontent.com/75536352/132244815-8aca532a-e32c-4e01-8fcc-46c16273cc28.png)v: i see it.

but prefix isnt visible.

Message: '&8[%alonsolevels_colored_level%&r&8]&r {player_prefix+}'

![image](https://user-images.githubusercontent.com/75536352/132244875-f3ca2221-1a08-45bb-b975-a47b672d1d66.png)

why?

**Comment by TheIntolerant** [comment]
Hey, so to use the prefixes from the ItemAdder addon, that will require you to install PlaceHolderAPI and use one of the placeholders in your format file shown <https://itemsadder.devs.beer/plugin-usage/placeholderapi>

Unfortunately I personally am not too familiar with item adder and kangarko is currently on vacation and banana I believe is unwell. So there may be some extended delay for a response from them if you are still having issues.

Anyhow, using the placeholder in the link above instead of {player_prefix+} should fix this, just remember chatcontrol uses {} not %% for placeholders.

---

In other words your format file should look something like this.

```
Message: '&8[{alonsolevels_colored_level}&r&8]&r {img_NAME}'
```

**Comment by ElBananaa** [comment]
Hey!
Thanks to gone-incognito for answering!
Apologize for the delay, as he said, i'm currently having health issues, so I can't spend much time on the computer.
Anyways, what he wrote should work, but please let us know if it doesnt.
